core-bundle/src/Twig/Interop/ContaoEscaperNodeVisitor.php

```diff
@@ -117,6 +117,23 @@ private function isEscaperFilterExpression(Node $node, string|null &$type = null
             return false;
         }
 
+        $doubleEncode = $node->getNode('arguments')->hasNode('double_encode') ? $node->getNode('arguments')->getNode('double_encode') : null;
+
+        if ($doubleEncode instanceof ConstantExpression && \is_bool($doubleEncode->getAttribute('value'))) {
+            $node->getNode('arguments')->removeNode('double_encode');
+
+            // Do not use the Contao escaper if `double_encode = true` is passed
+            if ($doubleEncode->getAttribute('value')) {
+                return false;
+            }
+        }
```

Comment:
@m-vo is it OK to implement the `double_encode` parameter for the `escape()` filter that way?
It is kind of a “virtual” parameter that way as it does not really exist in the escape method. I’m not sure if it would even be possible to add a real parameter to `escape()`?

Reply:
Probably yes as you are removing the node when it matches. Plus the whole interop stuff is meant to be removed in the future anyways.

You could in theory add more parameters via the `ContaoExtension`, because we are currently overwriting the `escape`/`e` filters to support our chunked text. But I would like to get rid of that, when there is a better extension point or way of implementing that in Twig...